gensim/models/wrappers/dtmmodel.py

```diff
@@ -283,26 +283,36 @@ def show_topics(self, num_topics=10, times=5, num_words=10, log=False, formatted
                 #     topic))
         return shown
 
-    def show_topic(self, topicid, time, num_words=50):
+    def show_topic(self, topicid, time, topn=50, num_words=None):
         """
         Return `num_words` most probable words for the given `topicid`, as a list of
         `(word_probability, word)` 2-tuples.
 
         """
+        if num_words is not None:  # deprecated num_words is used
+            logger.warn("The parameter num_words for show_topic() method would be deprecated in the updated version.\
```

Comment:
This would include the whitespace after `\` in the mesage.

It's better to split multi-line strings using `"abc" "dce"` (two strings next to each other, on different lines).